osu.Game/Screens/Select/BeatmapCarousel.cs

```diff
@@ -889,28 +889,10 @@ protected override void PerformSelection()
             }
         }
 
-        private class CarouselScrollContainer : OsuScrollContainer
+        private class CarouselScrollContainer : UserTrackingScrollContainer
         {
             private bool rightMouseScrollBlocked;
 
-            /// <summary>
-            /// Whether the last scroll event was user triggered, directly on the scroll container.
-            /// </summary>
-            public bool UserScrolling { get; private set; }
-
-            // ReSharper disable once OptionalParameterHierarchyMismatch 2020.3 EAP4 bug. (https://youtrack.jetbrains.com/issue/RSRP-481535?p=RIDER-51910)
```

Comment:
why was this removed? have you checked if this is resolved in rider EAP?

Reply:
yes it was. and it didn't trigger after removing so I guess the tracker is correct